manual/C/ch_Finance-Quote.xml

```diff
@@ -60,8 +60,9 @@ SYNOPSIS
 
     <note>
       <para>If you feel uncomfortable about performing any of these steps, please either email the &app;-user
-        mailing list (<email>gnucash-user@gnucash.org</email>) for help or come to the &app; IRC
-        channel on irc.gnome.org. You can also find more information in the &app; wiki at
+        mailing list (<email>gnucash-user@gnucash.org</email>) for help or come to the
+        <ulink url="&url-irc;">#&app; IRC channel</ulink>.
```

Comment:
Ich glaube im Fließtext `&app;`, aber in anderen Tags `&appname;` sieht besser aus?